src/Dfc.CourseDirectory.Functions/Dfc.CourseDirectory.Functions.csproj

```diff
@@ -6,10 +6,13 @@
   </PropertyGroup>
```

Comment:
I'm sure there was another package that needed to be upgraded to avoid a conflict warning...?

Reply:
I'm only getting the obsolete warnings in the build so I guess it's fine?

Reply:
It might've only been on a more up to date version of `Microsoft.Azure.WebJobs.Extensions.Storage`.